src/Mapping.elm

```diff
@@ -0,0 +1,463 @@
+module Mapping exposing (..)
+
+import Basics exposing (sqrt)
+import String exposing (toLower, trim)
+
+
+
+-- Exercise 1: Double the Numbers.
+-- Task: Write a function that doubles each number in a list of integers.
+
+
+doubleNumber : List Int -> List Int
+doubleNumber numbers =
+    List.map double numbers
+
+
+double : Int -> Int
+double number =
+    number * 2
+
+
+input : List Int
+input =
+    [ 1, 2, 3 ]
+
+
+output : List Int
+output =
+    doubleNumber input
```

Comment:
Please remove the sample inputs and outputs. They're just for you to test your solution, and not to include in your submission : 🤗🤗🤗🤗🤗